Index: .drone.yml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>kind: pipeline\ntype: docker\nname: build and publish\n\nsteps:\n  - name: publish\n    image: plugins/docker\n    settings:\n      username:\n        from_secret: docker_username\n      password:\n        from_secret: docker_password\n      tags:\n        - latest\n        - ${DRONE_COMMIT}\n      repo: gonzauge/spendspendspent\n\ntrigger:\n  branch:\n    - master\n  event:\n    - push\n\n---\nkind: pipeline\ntype: docker\nname: build\n\nsteps:\n  - name: publish\n    image: plugins/docker\n    settings:\n      dry_run: true\n\ntrigger:\n  branch:\n    exclude:\n      - master\n  event:\n    - push\n\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.drone.yml b/.drone.yml
--- a/.drone.yml	(revision 2401140afddd2f24a00bb65a299be2f1875f0e0c)
+++ b/.drone.yml	(date 1609995332737)
@@ -27,7 +27,7 @@
 name: build
 
 steps:
-  - name: publish
+  - name: build
     image: plugins/docker
     settings:
       dry_run: true
